Test/WebTestCase.php

```diff
@@ -574,18 +574,28 @@ public function loadFixtureFiles(array $paths = array(), $append = false, $omNam
 
         $files = $this->locateResources($paths);
 
-        // Check if the Hautelook AliceBundle is registered and if yes, use it instead of Nelmio Alice
-        $hautelookLoaderServiceName = 'hautelook_alice.fixtures.loader';
-        if ($container->has($hautelookLoaderServiceName)) {
-            $loaderService = $container->get($hautelookLoaderServiceName);
-            $persisterClass = class_exists('Nelmio\Alice\ORM\Doctrine') ?
-                'Nelmio\Alice\ORM\Doctrine' :
-                'Nelmio\Alice\Persister\Doctrine';
+        if (class_exists('Nelmio\Alice\Fixtures')) {
+            // nelmio/alice v2.*.*
+            // Check if the Hautelook AliceBundle is registered and if yes, use it instead of Nelmio Alice
+            $hautelookLoaderServiceName = 'hautelook_alice.fixtures.loader';
+            if ($container->has($hautelookLoaderServiceName)) {
+                $loaderService = $container->get($hautelookLoaderServiceName);
+                $persisterClass = class_exists('Nelmio\Alice\ORM\Doctrine') ?
+                    'Nelmio\Alice\ORM\Doctrine' :
+                    'Nelmio\Alice\Persister\Doctrine';
+
+                return $loaderService->load(new $persisterClass($om), $files);
+            }
 
-            return $loaderService->load(new $persisterClass($om), $files);
+            return Fixtures::load($files, $om);
+        } else {
+            $fidryLoaderServiceName = 'fidry_alice_data_fixtures.doctrine.persister_loader';
+            if ($container->has($fidryLoaderServiceName)) {
+                $container->get($fidryLoaderServiceName)->load($files);
```

Comment:
This should return the result of `->load($files)` so we have access to an array of fixtures like in Alice 2.x.